src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  justify-content: space-between;
+  gap: 46px;
+
+  @media (min-width: 1200px) {
+    padding-inline: 128px;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
```

Comment:
Your media query breakpoints seem to be a bit arbitrary. It's better to use commonly used breakpoints for small (576px), medium (768px), large (992px), and extra large (1200px) devices. This will ensure that your layout looks good on most devices.